client/stylesheets/abstracts/_colors.scss

```diff
@@ -61,3 +65,6 @@ $button-border: darken($button, 20%);
 // Muriel
 $muriel-gray-dark-300: #969ca1;
 $muriel-active: #bb4f10;
+
+// Onboarding
+$woocommerce-onboarding-background: #f6f6f6;
```

Comment:
Could we use `$muriel-gray-0` instead of this one?

Reply:
🤦‍♂Done!